test/unit/alignment/pairwise/fixture/semi_global_edit_distance_unbanded.hpp

```diff
@@ -35,7 +35,8 @@ static auto dna4_01 = []()
         // ---------ACGTACG-TA
         "AACCGGTTAACCGGTT"_dna4,
         "ACGTACGTA"_dna4,
-        seqan3::align_cfg::edit | seqan3::align_cfg::aligned_ends{seqan3::free_ends_first},
+        seqan3::align_cfg::method_global | seqan3::align_cfg::edit
+                                         | seqan3::align_cfg::aligned_ends{seqan3::free_ends_first},
```

Comment:
.. this

Reply:
Here I had to introduce a line break anyway

Reply:
The order does not matter, so I don;t see an issue?

Reply:
It doesn't matter too much, I simply like consistency within files.

Reply:
(But if I'm honest, it is bothering me a little :see_no_evil: )